3684/0/about_class_methods.rb

```diff
@@ -99,22 +101,22 @@ def test_you_can_define_class_methods_inside_the_class
 
   # ------------------------------------------------------------------
 
-  LAST_EXPRESSION_IN_CLASS_STATEMENT = class Dog
-                                         21
-                                       end
+  LASTEXPRESSIONINCLASSSTATEMENT = class Dog
+                                     21
+                                   end
 
   def test_class_statements_return_the_value_of_their_last_expression
-    assert_equal 21, LAST_EXPRESSION_IN_CLASS_STATEMENT
+    assert_equal 21, LASTEXPRESSIONINCLASSSTATEMENT
   end
 
   # ------------------------------------------------------------------
 
-  SELF_INSIDE_OF_CLASS_STATEMENT = class Dog
-                                     self
-                                   end
+  SELFINSIDEOFCLASSSTATEMENT = class Dog
+                                 self
+                               end
 
   def test_self_while_inside_class_is_class_object_not_instance
-    assert_equal true, Dog == SELF_INSIDE_OF_CLASS_STATEMENT
+    assert_equal true, Dog == SELFINSIDEOFCLASSSTATEMENT
```

Comment:
this change looks reaaaaaaaaaally weird. why?

Reply:
probably because rubocop 0.72.0 told me use CAPS, then hound told to use sneak_scream

Reply:
disable cop here. return the initial constant name

Reply:
solved